Project2015To2017/Project2015To2017.csproj

```diff
@@ -15,6 +15,7 @@
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <Version>1.8.1</Version>
     <AssemblyVersion>1.0.0.0</AssemblyVersion>
+    <LangVersion>7.3</LangVersion>
```

Comment:
C# 7.2 requirement for `ref struct`, that allows more CLR runtime optimizations and prevents storing the find results anywhere (since the results can change very quickly there is no point in storing the results except to introduce bugs).